app/components/visualizer/main-viz/main-viz.jade

```diff
@@ -1,2 +1 @@
-.main-viz.well.well-lg
-  p Main Viz
+svg#main-viz.well.well-lg
```

Comment:
That is an option too. Michael and I did this to avoid having to append an SVG in d3.